.github/workflows/helm-chart-release.yml

```diff
@@ -56,3 +59,14 @@ jobs:
           charts_dir: charts
         env:
           CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+
+      - name: Prepare GH pages readme
+        run: |
+          mkdir -p ./to-gh-pages
+          cp -f charts/kubewarden-controller/README.md ./to-gh-pages
+
+      - name: Deploy readme to GH pages
+        uses: JamesIves/github-pages-deploy-action@4.1.5
```

Comment:
We are using `peaceiris/actions-gh-pages@v3` GH action to do the very same thing inside of the docs and the policy-hub repos. It would be nice (but not mandatory), to use the same action everywhere.

Reply:
Oh, that's nice, thanks! Done in https://github.com/kubewarden/helm-charts/pull/37/commits/0a5b2b507cc925cdbebf65356cfce4ade5f0bb92.

Reply:
Great!